app/lib/analytics.coffee

```diff
@@ -74,9 +74,9 @@ logToGoogle = (eventData) =>
 ###
 This will log a user interaction both in the Geordi analytics API and in Google Analytics.
 ###
-logEvent = (type, related_id = '', user_id = User.current?.zooniverse_id, subject_id = ExperimentalSubject.current?.zooniverseId) =>
+logEvent = (type, related_id = '', subject_id = ExperimentalSubject.current?.zooniverseId) =>
```

Comment:
I think your method calls would be cleaner if you used a params object, rather than passing them all separately. Like so:

```
logEvent = (params = {}) ->
  DEFAULT_PARAMS =
    type: null
    related_id: ''
    subject_id: ExperimentalSubject.current?.zooniverseId

  params = Object.assign params, DEFAULT_PARAMS
```